forEach.js

```diff
@@ -0,0 +1,16 @@
+let family = ["Sofi","Hen", "Len"];
+
+function forEch(arr, callback){
+  for(let i = 0; i < arr.length; i++){
+    let item = (arr[i]);
+    let index = (i);
+    let array = (arr);
+    callback(item, index, array);
+  }
+};
+
+function cb(it, ind, ar){
+  console.log(it, ind, ar)
+};
```

Comment:
Name your variables whatever you want. They will not been seen from the outer scope

```suggestion
function cb(item, index, arr){
  console.log(item, index, arr)
};
```